spec/models/scheme_spec.rb

```diff
@@ -83,4 +83,16 @@
       s.details.should == { "foo" => "foo", "bar" => "bar" }
     end
   end
+
+  describe "area codes returned from imminence" do
+    it "should blacklist some area codes" do
```

Comment:
'blacklist' here is potentially confusing now that this is using a whitelist - how about something like 'should only use whitelisted area types'?